contracts/test/fuzz/JBOperatorStore.t.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.6;
+
+import 'ds-test/test.sol';
+import 'forge-std/Vm.sol';
+
+import {JBOperatorStore, PERMISSION_INDEX_OUT_OF_BOUNDS} from '../../../contracts/JBOperatorStore.sol';
+import {JBOperatorData} from '../../../contracts/structs/JBOperatorData.sol';
+
+contract JBOperatorStoreTest is DSTest {
```

Comment:
Don't have a good enough handle on the code base to write meaningful tests right now, but included this as an example incase it is useful (although prob not testing anything useful)